src/pipeline.cc

```diff
@@ -31,8 +31,8 @@ Pipeline::ShaderInfo::~ShaderInfo() = default;
 
 const char* Pipeline::kDefaultColorBufferFormat = "B8G8R8A8_UNORM";
 const char* Pipeline::kDefaultDepthBufferFormat = "D32_SFLOAT_S8_UINT";
```

Comment:
VkRunner assumes that there is no depth buffer in default.
I agree with the idea that no depth buffer in default would be less unexpected behavior because some rendering result would be different because of depth buffer.

Is there any special reason for this?
(sorry for not claiming this when you wrote Amber script doc)

Reply:
So, you don't think we should have a depth buffer in amber either? I can remove it in VkScript, but do you think the general user would expect there to be a depth buffer no no depth buffer? @dneto0 do you have any idea what the expected would be?

Reply:
Updated VkScript to not generate a depth buffer by default. Left amberscript to generate one. We can decide if that's the right route for amberscript in the future.

Reply:
I expect no depth buffer by default.

Reply:
K, this CL is already updated. I'll make the amber change in a separate CL as it's unrelated to the rest of this CL.

Reply:
Agree depth/stencil buffer should not be used by default.
Vkrunner says about depthstencil directive:   "Without it no depth-stencil buffer will be created."

Reply:
Yup, I've done that for VkScript already. Will handle updating AmberScript to not have a default depth buffer in a followup.